src/System.Private.ServiceModel/src/System/IdentityModel/Selectors/X509CertificateValidator.cs

```diff
@@ -13,7 +13,9 @@ namespace System.IdentityModel.Selectors
 {
     public abstract class X509CertificateValidator
     {
+        private static X509CertificateValidator s_peerTrust;
```

Comment:
Reviewers -- this code was copied verbatim from the full framework with only these changes:
- rename of fields to be consistent with new conventions
- use of X509Store rather than IdentityModel's X509CertificateStore.
- SR.Format